README.md

```diff
@@ -1462,6 +1462,48 @@ jobs:
           publish-summary: false
 ```
 
+### Automatic PR number & URL detection
+
+The PR number and URL can be automatically detected if you pass `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` via the workflow `env`. When set,
+this value enables the Action to perform additional logic that grabs the related PR number and URL (if they exist) and sets them in the
+environment variables `CYPRESS_PULL_REQUEST_ID` and `CYPRESS_PULL_REQUEST_URL`, respectively.
+
+Example workflow using the new variables:
+```yml
+name: Example echo PR number and URL
+on: push
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cypress run
+        uses: cypress-io/github-action@v5
```

Comment:
It might be helpful to:
* Prefix this section with "If you are recording your runs to Cypress Cloud, ...", as there's no reason to fetch PR info they aren't going to be recording the run.
* Update this example to include the `record`-related options for the action (like [here](https://github.com/cypress-io/github-action#record-test-results-on-cypress-cloud))
* Add a link to [our existing docs](https://docs.cypress.io/guides/continuous-integration/introduction#CI-Build-Information) for the `CYPRESS_PULL_REQUEST_*` envs for user reference.